mender-deb-package

```diff
@@ -135,6 +135,9 @@ prepare_recipe() {
     if [ "${debian_recipe}" = "debian-2.1.x" ]; then
       # Skip for 2.1.x, special handling in debian/rules
       true
+    elif [ "${debian_recipe}" = "debian-master" ]; then
+      # skip
+      true
```

Comment:
Shouldn't this have a task to remove? It's ok that we don't need it now, since we have no daemon mode, but we shouldn't leave it like this.